airflow/providers/google/cloud/example_dags/example_dlp.py

```diff
@@ -177,3 +178,31 @@
     )
     # [END howto_operator_dlp_delete_job_trigger]
     create_trigger >> update_trigger >> delete_trigger
+
+DEIDENTIFY_CONFIG = {
+    "info_type_transformations": {
+        "transformations": [
+            {
+                "primitive_transformation": {
+                    "replace_config": {"new_value": {"string_value": "[deidentified_number]"}}
+                }
+            }
+        ]
+    }
+}
+
+with models.DAG(
```

Comment:
Can you add system tests? We have a rule that each DAG for GCP services should have system tests to verify its correctness. 
https://github.com/apache/airflow/blob/master/TESTING.rst#airflow-system-tests

Reply:
Sure - done.